lib/core-integration/src/Test/Integration/Scenario/API/Transactions.hs

```diff
@@ -1025,26 +1040,373 @@ spec = do
         expectResponseCode @IO HTTP.status404 r
         expectErrorMessage (errMsg404NoWallet $ w ^. walletId) r
 
+    it "TRANS_LIST_01 - Can list Incoming and Outgoing transactions" $ \ctx -> do
+        (wSrc, _, _) <- fixtureWalletManyTxs ctx [1]
+        let (feeMin, feeMax) = ctx ^. feeEstimator $ TxDescription
+                { nInputs = 1
+                , nOutputs = 1
+                }
```

Comment:
I don't think asserting fees is really relevant for that unit test. If we're looking into behavior of the list transaction endpoint, we should reduce the noise around and only checks what's relevant to listing transactions.

Reply:
It was just to assert for the amount of Outgoing transaction, but OK, removing for the sake of making the test clearer.